cluster-autoscaler/utils/kubernetes/factory.go

```diff
@@ -27,12 +27,31 @@ import (
 	"k8s.io/klog"
 )
 
+const (
+	// Rate of refill for the event spam filter in client go
+	// 1 per event key per 5 minutes.
+	defaultQPS = 1. / 300.
```

Comment:
Why are those called `default*` if they cannot be overriden with something non-default?

Reply:
Default just means preselected. For eg. there were defaults in client-go that couldn't be overridden. 

I'll change it if you feel strongly about this.